components/cli/pkg/commands/image/build.go

```diff
@@ -331,11 +339,17 @@ func createArtifactsZip(cli cli.Cli, artifactsZip, projectDir, fileName string)
 			return fmt.Errorf("error occured while copying the %s, %v", ballerinaLocalRepo, err)
 		}
 	}
-	isTestDirExists, _ := util.FileExists(zipTests)
-	folders := []string{filepath.Join(cli.FileSystem().WorkingDirRelativePath(), artifacts), filepath.Join(cli.FileSystem().WorkingDirRelativePath(), src)}
+	isTestDirExists, _ := util.FileExists(filepath.Join(balParent, constants.ZipTests))
+	folders := []string{artifacts, src}
```

Comment:
Check if unit tests are passing. Better to get the relative path of folders (artifacts, src) from cli context.

Reply:
Fixed paths.